test/client/karma.spec.js

```diff
@@ -462,6 +465,26 @@ describe('Karma', function () {
       clock.tick(10)
     })
 
+    it.only('should not navigate the client to return_url if not benign', function () {
```

Comment:
nit: remove `.only`

Reply:
You're right in that I need to remove `.only` but wrong in that it is a nit. I would say it is mandatory :) 👍

Reply:
Done.